app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager.rb

```diff
@@ -31,6 +31,12 @@ class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager < ManageI
            :autosave    => true,
            :dependent   => :destroy
 
+  has_many :placement_groups,
+           :foreign_key => :ems_id,
+           :class_name  => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::PlacementGroup",
+           :autosave    => true,
+           :dependent   => :destroy
+
```

Comment:
You don't have to declare this here since it is already in the base ExtManagementSystem class here https://github.com/ManageIQ/manageiq/pull/21807/files#diff-8d286308b75ae7dc8a266753732b0c1709d33293518c479765e34cadf8207910R90